resources/user-agents/browsers/firefox-on-ios/firefox-ios-15-on.json

```diff
@@ -1,6 +1,6 @@
 {
   "division": "Firefox #MAJORVER#.#MINORVER# for iOS",
-  "versions": [125, 124, 123, 122, 121, 120, 119, 118, 117, 116, 115, 114, 113, 112, 111, 110, 109, 108, 107, 106, 105, 104, 103, 102, 101, 100, 39, 38, 37, 36, 35, 34, 21, 20, 19, 18, 17, 16, 15],
+  "versions": [135, 134, 133, 132, 131, 130, 129, 128, 127, 126, 125, 124, 123, 122, 121, 120, 119, 118, 117, 116, 115, 114, 113, 112, 111, 110, 109, 108, 107, 106, 105, 104, 103, 102, 101, 100, 39, 38, 37, 36, 35, 34, 21, 20, 19, 18, 17, 16, 15],
```

Comment:
@mimmi20 given the latest Chrome is 126, and Firefox 127, how many of these additional "future" versions do we actually want to add? What if Chrome 128 isn't released for a long time, but actually has a new UA format etc? Do we consider this problematic, the addition of anticipated future versions? I suppose there is a trade-off between maintenance needed for new versions, and accuracy of UA identification 🤷

Otherwise, this PR lgtm.

Reply:
I did consider this.  But with release cycles being every 6 weeks and canary already being 128, I felt the balance of having to submit a PR every 6 to 12 weeks vs a fixup if the UA format changed was worth it.  The Chrome format for example looks to have been stable for the last 60+ versions so felt it was a fair gamble.  Additionally I wasn't sure why the projects release cycle was, as the last version appears to be around 6 months old I tried to account for that too.

Reply:
We have to see, what the future of useragents will be. When asking about future versions, it is also questionable how long the old versions or old devices should be kept.

Reply:
> Additionally I wasn't sure why the projects release cycle was, as the last version appears to be around 6 months old I tried to account for that too. 

I release it whenever someone makes a meaningful change (i.e. not just dependencies updated).

Hardly anyone submits PRs like this - they usually just open an issue and a few weeks later complain it isn't fixed 😂 so your PR is extremely appreciated - so thank you @allan-reynolds for being proactive! ❤️